### PR TITLE
changes for edit and delete

### PR DESCRIPTION
**Pull request by Prachiashi**

### CODE REVIEW

**Comment by Prachiashi** [comment]
aaaaaa

**Review by rjspykar — CHANGES_REQUESTED** [review]
Implement delete functionality to remove row from delete button click

File: lib/Bean/TODO.dart [hunk 1]
```diff
@@ -68,6 +68,30 @@ class TODO implements Comparable<TODO> {
...
+    final response = await client.delete(
+      Uri.parse("http://localhost:8080/deleteTODO"),
+      headers: {
+        "Accept": "application/json",
+        "Content-Type": "application/json"
+      },
+      body: jsonEncode(todo.toJson()),
+    );
+
+    if (response.statusCode == 204) {
```
Comment (rjspykar) [inline lib/Bean/TODO.dart:84]: Why have you put statusCode as 204 ??

File: lib/StatefulWidget/todo_list.dart
Comment (rjspykar) [inline lib/StatefulWidget/todo_list.dart:1]: Why is this file deleted ?

File: lib/Bean/TODO.dart (hunk 1 above)
Comment (rjspykar) [inline lib/Bean/TODO.dart:84]: What is the reason behind putting statuscode as 204?

**Review by rjspykar — CHANGES_REQUESTED** [review]
i have some doubts. please explain it and send pr again.

**Comment by rjspykar** [comment]
First PR Merge

**Pull request merged**